pipelines/release.yml

```diff
@@ -78,8 +81,10 @@ jobs:
       echo "## Building ${IMAGE_NAME}:$(NNI_RELEASE) ##"
       docker build --build-arg NNI_RELEASE=$(NNI_RELEASE) -t ${IMAGE_NAME} .
       docker tag ${IMAGE_NAME} ${IMAGE_NAME}:$(NNI_RELEASE)
-      docker push ${IMAGE_NAME}
       docker push ${IMAGE_NAME}:$(NNI_RELEASE)
+      if [[ $(build_type) != 'rc' ]]; then
```

Comment:
what if `$(build_type) == 'release'`, `rc` means `release`?

Reply:
This line means, if build type is "rc", do not tag the image as "latest".